2016-09-07 17:04:33,227 manager INFO 1.1.0
2016-09-07 17:04:33,227 manager INFO Copyright (C) 2003-2010
2016-09-07 17:04:33,227 manager INFO   Noriaki Ando
2016-09-07 17:04:33,227 manager INFO   Intelligent Systems Research Institute, AIST
2016-09-07 17:04:33,227 manager INFO Manager starting.
2016-09-07 17:04:33,227 manager INFO Starting local logging.
2016-09-07 17:04:33,232 manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2016-09-07 17:04:35,240 ManagerServant INFO Master manager not found
2016-09-07 17:04:35,240 manager WARNING Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\Manager.py", line 1900, in initManagerServant
    reffile = file(self._config.getProperty("manager.refstring_path"),'w')
IOError: [Errno 2] No such file or directory: '/var/log/rtcmanager.ref'

2016-09-07 17:04:35,240 LocalServiceAdmin INFO All the local services are enabled.
2016-09-07 17:04:35,243 manager INFO Component type conf file: outTemp.conf loaded.
2016-09-07 17:04:35,244 ec_base INFO setRate(1000000.000000) done
2016-09-07 17:04:35,244 ec_base INFO setRate(1000.000000) done
2016-09-07 17:04:35,246 rtobject INFO 1 execution context was created.
2016-09-07 17:04:35,247 rtobject INFO Initial active configuration set is default-set.
2016-09-07 17:04:43,197 SdoServiceAdmin WARNING No available SDO service in the factory: 
2016-09-07 17:04:43,197 SdoServiceAdmin ERROR type  already exists.
2016-09-07 17:04:43,197 SdoServiceAdmin WARNING No available SDO service in the factory: IDL:OpenRTM/ComponentObserver:1.0
2016-09-07 17:04:43,197 SdoServiceAdmin ERROR type 009fcfe9-cc91-4411-9c82-fe926013192e already exists.
2016-09-07 17:04:50,017 rtobject ERROR Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\RTObject.py", line 1560, in on_execute
    ret = self.onExecute(ec_id)
  File "outTemp.py", line 182, in onExecute
    self._d_origin_Temp.read()
AttributeError: 'TimedDouble' object has no attribute 'read'

2016-09-07 17:10:16,519 ec_worker ERROR State of the RTC is not ACTIVE_STATE.
2016-09-07 17:10:19,154 ec_worker ERROR State of the RTC is not ACTIVE_STATE.
2016-09-07 17:10:19,168 ec_worker WARNING ExecutionContext is already stopped.
2016-09-07 17:10:19,168 ec_base ERROR Invoking on_shutdown() for each RTC failed.
